### ISSUE TITLE
InfoPlist.strings files are not updated due to use of ucs2 encoding

### ISSUE THREAD

**Issue by arudnev** [evshiron/nwjs-builder-phoenix#29]
It seems that replacements that are supposed to happen in Builder.fixMacMeta are not happening, leaving all those InfoPlist.string files with original values.

Assuming that path variable in that operation points to something like ${name}.app/Contents/Resources/en.lproj/InfoPlist.strings here is what is what would be returned from readFileAsync, which results into regexp replacements being skipped:

```
> require('fs-extra-promise').readFileSync(path, {encoding: 'ucs2'})
```
```
'䙃畂摮敬楄灳慬乹浡⁥‽渢橷≳਻䙃畂摮敬敇䥴普卯牴湩⁧‽渢橷⁳㠵〮㌮㈰⸹ㄸ‬潃祰楲桧⁴〲㜱吠敨䌠牨浯畩⁭畁桴牯ⱳ丠⹗獪挠湯牴扩瑵牯ⱳ丠摯⹥獪‮汁⁬楲桧獴爠獥牥敶⹤㬢䌊䉆湵汤乥浡⁥‽渢橷≳਻华潃瑮捡獴獕条䑥獥牣灩楴湯㴠∠敄慴汩⁳牦浯礠畯⁲潣瑮捡獴挠湡栠汥⁰潹⁵楦汬漠瑵映牯獭洠牯⁥畱捩汫⁹湩䌠牨浯畩⹭㬢上䡓浵湡敒摡扡敬潃祰楲桧⁴‽䌢灯特杩瑨㈠㄰‷桔⁥桃潲業浵䄠瑵潨獲‬坎樮⁳潣瑮楲畢潴獲‬潎敤樮⹳䄠汬爠杩瑨⁳敲敳癲摥∮਻'
```

Most likely simple change from ucs2 to utf-8 or something like that in calls to readFileAsync / writeFileAsync would fix the issue:

```
> require('fs-extra-promise').readFileSync(path, {encoding: 'utf-8'})
```
```
'CFBundleDisplayName = "nwjs";\nCFBundleGetInfoString = "nwjs 58.0.3029.81, Copyright 2017 The Chromium Authors, NW.js contributors, Node.js. All rights reserved.";\nCFBundleName = "nwjs";\nNSContactsUsageDescription = "Details from your contacts can help you fill out forms more quickly in Chromium.";\nNSHumanReadableCopyright = "Copyright 2017 The Chromium Authors, NW.js contributors, Node.js. All rights reserved.";\n'
```

**Comment by evshiron** [comment]
Hmm? When I used `utf-8` it didn't replace (if I remember right it was for NW.js 0.14.7), but printed well because for ASCII characters the additional `0x00` wouldn't make a difference. Finally I used Hex Fiend and confirmed it's UTF-16LE, and it did work with `ucs2` at that time. Is this an issue associated with different NW.js versions?

I also have a test for it in `/test/Builder.js`, could you clone this repo, `npm install` and `npm test ./test/Builder.js`, and see if the macOS build contains the correct copyright `"copyright"`?

**Comment by evshiron** [comment]
Oh, I just tested with the latest version of NW.js, they are UTF-8 now. I am working on it.

**Comment by evshiron** [comment]
Issue is fixed in `nwjs-builder-phoenix` 1.10.1, please upgrade and check it out.
